libs/community/langchain_community/document_loaders/quip.py

```diff
@@ -50,6 +58,7 @@ def load(
         include_all_folders: bool = False,
         include_comments: bool = False,
         include_images: bool = False,
+        keep_html_format: bool = False,
```

Comment:
if this is False then a new dependency is required right (beautifulsoup)? since that's a breaking change, can we default this to True?

Reply:
I apologize for the long delay in my response. I realize that our communication has not been very efficient. After receiving your comments last time, I forgot to follow up on this pull request. 

Hopefully this PR can be merge soon. Thanks.